lib/getstats.js

```diff
@@ -316,6 +323,38 @@ function chromeGetTrackStats(peerConnection, track) {
   });
 }
 
+/**
+ * Get the standardized statistics for a particular MediaStreamTrack in Edge.
+ * @param {RTCPeerConnection} peerConnection
+ * @param {MediaStreamTrack} track
+ * @returns {Promise.<StandardizedTrackStatsReport>}
+ */
+function edgeGetTrackStats(peerConnection, track) {
+  return waitForEdgeIceCompletion(peerConnection).then(function() {
```

Comment:
Can you include a note linking to some Edge documentation or bug that describes why we must wait?